egs/voxpopuli/ASR/local/preprocess_voxpopuli.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python3
+# Copyright    2023  Xiaomi Corp.        (authors: Yifan Yang)
```

Comment:
Maybe you need to update the author information?

Reply:
ok, done here and also in other .py scripts...